app/frontend/test/setup.js

```diff
@@ -0,0 +1,5 @@
+console.error = message => {
+  throw new Error(
+    `Failing due to console.error while running test!\n\n${message}`
```

Comment:
So this overrides what console.error does and throws a real error in the test suite's context? Can you point to the docs that this is what to do pls in the PR?

Reply:
> So this overrides what console.error does and throws a real error in the test suite's context?

Correct

> Can you point to the docs that this is what to do pls in the PR?

There's nothing in the docs about this. I've seen a few people do workarounds for this but they're all variations on this theme.

Thinking about it I think it might make more sense to do this in a beforeEach block, use a spy instead, and restore it in an afterEach block. I'll have a go and see if that works.

Reply:
If yes it might be worth raising an issue with Vitest developers requesting this functionality (if there isn't already an issue), seeing as they are saying it should be a drop-in replacement for Jest.